src/algos/simba_algo.py

```diff
@@ -33,6 +33,12 @@ def init_client_model(self, config):
         if config["model_name"] == "resnet18":
             model = models.resnet18(pretrained=config["pretrained"])
             model = nn.Sequential(*nn.ModuleList(list(model.children())[:config["split_layer"]]))
+        elif config["model_name"] == "resnet34":
```

Comment:
Since the second line is same for all of them, can you create the model inside `if` and assign this model `if config["model_name"].startswith("resnet")`

Reply:
Yeah, will update this